include/mbedtls/ssl_internal.h

```diff
@@ -1229,4 +1229,184 @@ void mbedtls_ssl_buffering_free( mbedtls_ssl_context *ssl );
 void mbedtls_ssl_flight_free( mbedtls_ssl_flight_item *flight );
 #endif /* MBEDTLS_SSL_PROTO_DTLS */
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+/**
+* \brief   The data structure holding the cryptographic material (key and IV)
+*          used for record protection in TLS 1.3.
+*/
+typedef struct mbedtls_ssl_key_set {
+    /*! The key for client->server records. */
+    unsigned char clientWriteKey[ MBEDTLS_MAX_KEY_LENGTH ];
```

Comment:
Style: I don't think we do camelCase. (Multiple occurrences.)

Reply:
Oh, right. I'll fix it, and pay attention to this when reworking further parts of the prototype for upstreaming.